### PR TITLE
link to hoc committee pages from partner page

### PR DESCRIPTION
**Pull request by tanyaparker**
**partners page**
image

**FAQ**
image

**Footer**
image

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -281,7 +281,7 @@
...
   hoc_faq_behind_hoc_q: 'Who is behind the Hour of Code?'
-  hoc_faq_behind_hoc_a: "The Hour of Code is organized by <a href='https://code.org'>Code.org</a>, a public 501c3 non-profit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented students of color. An <a href='%{codeorg_url}/about/partners'>unprecedented coalition of partners</a> have come together to support the Hour of Code, too &mdash; including Microsoft, Apple, Amazon, Boys and Girls Clubs of America and the College Board."
+  hoc_faq_behind_hoc_a: "The Hour of Code is driven by the Hour of Code and Computer Science Education Week <a href='%{partner_url}'>Advisory and Review Committees</a> as well as an unprecedented coalition of partners that have come together to support the Hour of Code &mdash; including Microsoft, Apple, Amazon, Boys and Girls Clubs of America and the College Board."
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:284]: plural committees?

File: pegasus/sites.v3/hourofcode.com/public/partners.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
 title: <%= hoc_s(:title_partners) %>
 ---
 
-<%= view :signup_button %>
+The Hour of Code is driven by the Hour of Code and Computer Science Education Week Advisory and Review Committees. Representatives from K-12, academia, nonprofits, for-profits, and international organizations compose the [Advisory Committee](<%= resolve_url('/advisory-committee') %>) that has guided the strategy of the Hour of Code campaign.
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/partners.md]: no hyphen in nonprofit, but hyphen in for-profits?

File: pegasus/sites.v3/hourofcode.com/public/partners.md [hunk 3]
```diff
@@ -2,7 +2,11 @@
...
+
+The [Review Committee](<%= resolve_url('/review-committee') %>) is composed of 15 educators across K-12 grade bands teamed up to assess and recommend activities using the Advisory Committee's rubric. These educators reviewed student-led tutorials and teacher-led lesson plans submitted from hundreds of tutorial partners, evaluating these activities on metrics such as their educational value, fun and engagement, and appeal to a diverse set of students.
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/partners.md]: submitted by, perhaps?

File: pegasus/sites.v3/hourofcode.com/public/partners.md [hunk 4]
```diff
@@ -2,7 +2,11 @@
...
+
+Both committee's work and dedication has contributed to the success of the Hour of Code and its goal that any student of any age can try an introduction to computer science.
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/partners.md]: committees'

Comment (breville) [reply pegasus/sites.v3/hourofcode.com/public/partners.md]: computer science lower case?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:284]: They are two different committees.

File: pegasus/sites.v3/hourofcode.com/public/partners.md (hunk 2 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/public/partners.md]: Seems right to me.

http://www.merriam-webster.com/dictionary/for-profit
http://www.merriam-webster.com/dictionary/nonprofit

File: pegasus/sites.v3/hourofcode.com/public/partners.md (hunk 3 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/public/partners.md]: Agree.

File: pegasus/sites.v3/hourofcode.com/public/partners.md (hunk 4 above)
Comment (tanyaparker) [reply pegasus/sites.v3/hourofcode.com/public/partners.md]: yes to s' 

I think Hadi mentioned this once that CS should be capitalized when it's in a title but just talking about the subject is fine as lower case.

**Comment by tanyaparker** [comment]
Circle failure is Big Game

**Pull request merged**